File: polars/polars.py
import pandas as pd

df_csv = pd.read_csv("./data/df.csv")
# df_parquet = pd.read_parquet("./data/df.parquet")

print(
    df_csv[
        df_csv.InsurerLabel == "andsafe"]
    .groupby(by="TagLabel")
    .sum()
    [["TagLabel", "Total", "Year", "Quartal", "Week"]]

)

File: polars/polars.ts
import pl from "nodejs-polars";

const df = pl.readParquet("./data/df.parquet");

const df2 = df.select(
  pl
    .col("Week")
    .filter(pl.col("InsurerLabel").eq(pl.lit("andsafe")))
    .sum()
);

console.log(df2.head());
